xbmc/filesystem/MythFile.cpp

```diff
@@ -518,7 +518,7 @@ bool CMythFile::SkipNext()
 {
   HandleEvents();
   if(m_recorder)
-    return m_dll->recorder_is_recording(m_recorder) > 0;
+    return m_dll->recorder_is_recording(m_recorder) > (void *)0;
```

Comment:
I've always preferred the implicit conversion of pointers to boolean. E.g.;
```cpp
return !!m_dll->recorder_is_recording(m_recorder);
```
or
```c++
if (CGUIDialogNumeric::ShowAndGetTime(time, heading ? heading : g_localizeStrings.Get(21420)))
```
but this works too ^^